apps/ejabberd/src/ejabberd_ctl.erl

```diff
@@ -64,7 +64,7 @@
 
 -define(ASCII_SPACE_CHARACTER, $\s).
 -define(PRINT(Format, Args), io:format(lists:flatten(Format), Args)).
--define(TIME_HMS_FORMAT, "~B:~2.10.0B:~2.10.0B").
+-define(TIME_HMS_FORMAT, "~B days ~B hours ~B minutes ~B secs").
```

Comment:
I think output like `X days 12:30:11` would be enough. Also the formatting `~2.10` should be preserved.